swarm/api/act.go

```diff
@@ -0,0 +1,319 @@
+package api
```

Comment:
Maybe it is nitpicking, but I would create a new `act` package, and not just a file. That way we can make clear what is the api of the act feature